ticketutil/jira.py

```diff
@@ -91,35 +91,38 @@ def _verify_ticket_id(self, ticket_id):
                 logger.error(e)
             return False
 
-    def create(self, summary, description, **kwargs):
+    def create(self, summary, description, type, **kwargs):
         """
         Creates a ticket.
-        The required parameters for ticket creation are summary and description.
+        The required parameters for ticket creation are summary, description and issue type.
         Keyword arguments are used for other ticket fields.
         :param summary: The ticket summary.
         :param description: The ticket description.
+        :param type: The ticket issue type.
         :return: self.request_result: Named tuple containing request status, error_message, and url info.
         """
         error_message = ""
         if summary is None:
             error_message = "summary is a necessary parameter for ticket creation"
         if description is None:
             error_message = "description is a necessary parameter for ticket creation"
+        if type is None:
+            error_message = "issue type is a necessary parameter for ticket creation"
```

Comment:
Let's change 'issue type' here to simply 'type', to not confuse the user into thinking they need to pass in a parameter called 'issue type'.